### ISSUE TITLE
Allow filtering of sign up notifications 

### ISSUE THREAD

**Issue by mdiluz** [LucasGGamerM/moshidon#468]
**Is your feature request related to a problem? Please describe.**
As a moderator on a popular server my notification feed can be quite full of sign up notifications, and currently there's no way to filter them out.

**Describe the solution you'd like**
Add a notification filter on the notification tab and notification settings for sign up notifications.

**Describe alternatives you've considered**
Alternative here is probably just to use FediLab or another app that has the filter

**Additional context**
Let me know if you need more details!

**Comment by LucasGGamerM** [comment]
Hmmm, this may need more thought on how to show the option... I will consider this thoroughly.

**Comment by mdiluz** [comment]
Yeah had a look myself to see if I could contribute it but it looked a little harder than just a couple of file additions
